packages/core/EventManager.ts

```diff
@@ -210,7 +217,7 @@ export default class EventManager {
     const results: Array<EventResult<Event>> = [];
     // If and only if event type is a dedicated meeting, update the dedicated video meeting.
     if (isDedicated) {
-      const result = await this.updateVideoEvent(evt, booking);
+      const result = await this.updateVideoEvent(evt, booking as unknown as PartialBooking);
```

Comment:
```suggestion
      const result = await this.updateVideoEvent(evt, booking);
```

I'm sorry @joeauyeung I can't allow you to do that.

![](https://media1.giphy.com/media/CdY6WueirK8Te/200.gif)